### ISSUE TITLE
PHPBB 3.1

### ISSUE THREAD

**Issue by JzTurrini** [ComBoot/ComBoot#24]
Hi, nice job :)

This style works in phpbb3.1?

**Comment by TheZoker** [comment]
Thank you :smile: 
I`m working on a version for phpBB3.1

If you want to get an email, if the version is available, you can subscribe to the update newsletter:
http://eepurl.com/89FUv or here http://comboot.de/changelog/

If you have any question or found a issue, let me know :)

**Comment by JzTurrini** [comment]
Okay, I signed up.

I haven't tested fully the style. I am developing a new version of my RPG forum in phpbb3.1, reprogramming modules and etc. 

I have in mind to use your style as a basis for it, creating it with SASS the possibility of different colors and fonts, depending on the forum that the player is. And create new templates to my modifications.

Your style fits because you use the font-awesome, bootstrap3 and be responsive, which is great.

PS: Sorry by my terrible english.

**Comment by TheZoker** [comment]
I understand you english very well :)

So you are using the sass version of bootstrap?
I may change my style file to less in the future, for easy customization

Do you have any idea for new features?

If you need any help, just tell me :)

**Comment by JzTurrini** [comment]
In fact, I made a fork of your project because I want to convert their styles to SASS. Do not intend to use the version of bootstrap3 to sass, I will use your full css.

I Use the SASS more to facilitate the creation of mixins and a color map, you know? I've never used LESS but it is easy to convert for SASS also.

For my RPG Harry Potter, I will re-create, in essence:
- Module of Bank;
- Module of Player Data;
- Module of Spells;
- Module of Attributes;
- Module of Skills;
- Chatbox (this I will adapting other code);
- Module of Rank Player;
- Module of Students Bulletin;
- Module of companies;
- Module of Quidditch;
- Module of Duels;
- Module of Registration Posts;
- Module of Virtual Stores;
- Module of Sorting Hat;
- User bar tools.
- Module of owl;

I will change, particularly, overall_header, overall_footer, and Viewtopic ViewForum to use my modules. These are very specific tools of my RPG I want to, after 11 years, provide freely in phpbb3.1. In these cases I intend to work the styles following the concept of ComBoot.

I know, for sure, I will have ideas for new features when developing on inside of your style and for sure I will pass on the ideas for you.

Again, congratulations for the work.

PS: My RPG is: http://www.zonkos.com.br/magia/index.php
I'm know, the actual style is awful. It's a is very old style.

**Comment by TheZoker** [comment]
You should wait until the phpbb3.1 version ready, so you can start with a working platform and then make your changes....

You can read more about it here:
http://comboot.de/blog/dev/phpbb-3-1/

Let me know, if there is anything else I can help you with :)

**Comment by erhan-** [comment]
I will base the next update of my forum on this style. 

Danke.

**Comment by TheZoker** [comment]
Thank you :)
I hope the version for 3.1 will be finished this month :)

More here:
http://comboot.de/blog/dev/phpbb-3-1/

**Comment by erhan-** [comment]
I know the post :D No need to be quick for me, it has no priority for me atm. ;)

**Comment by sim2github** [comment]
To install to phpbb 3.1.3 add to `style.cfg` this lines:

``` php
style_version = 3.1.3
phpbb_version = 3.1.3
```

**Comment by TheZoker** [comment]
Yes you can use it with that line, but many things are not compatible e.g. notifications, extensions ...

**Comment by erhan-** [comment]
@sim2github 
Thanks, didn't know that.

@TheZoker 
Any plans on the release?

**Comment by TheZoker** [comment]
I hope at the end of march. Its a lot of work and I have many other projects, but I will do my best :)

**Comment by Cyral** [comment]
Is a version of the 3.1 release available anywhere yet or are you still working on it?

**Comment by TheZoker** [comment]
Its still in development. 
More information here: http://comboot.io/blog/dev/phpbb-3-1/
and current status here: https://trello.com/b/qDGdzDzU/comboot-3-1

**Comment by erhan-** [comment]
Thanks. I think trello will be sufficient to see how far the development is at the moment.

**Comment by TheZoker** [comment]
Closed beta is available: http://comboot.io/blog/beta/beta-3-1/

**Comment by TheZoker** [comment]
So if anyone of you want to be part of the closed beta, just let me know :)
The most things are fixed now, there are only some little open issues.

**Comment by Galixte** [comment]
Hi @TheZoker,

i would to test it, i’m already signed up “Beta” on your website, and now what's happening ?

**Comment by Galixte** [comment]
thanks !

**Comment by juliengresse** [comment]
Hi Comboot !

I can't find the beta release for phpBB 3.1.
Is there a repository with actual code to test it ?
